atest/robot/libdoc/doc_format.robot

```diff
@@ -44,12 +44,31 @@ Format in XML
     ROBOT    --docfor RoBoT   DocFormatHtml.py
     HTML     ${EMPTY}         DocFormatHtml.py
 
+Format in XML:HTML
+    [Template]    Test Format in XML:HTML
+    HTML    --format xMl:hTML            DocFormat.py
+    HTML    --docfor RoBoT -f XML:HTML   DocFormatHtml.py
+    HTML    -F ROBOT --format xml:html   DocFormat.py
+    
 Format from XML spec
     [Template]    NONE
     Run Libdoc    -F HTML ${TESTDATADIR}/DocFormat.py ${OUTXML}
     Copy File    ${OUTXML}    ${OUTPREFIX}-2.xml
     Test Format In XML    HTML    lib=${OUTPREFIX}-2.xml
 
+Format from XML:HTML spec
+    [Template]    NONE
+    Run Libdoc    -F ROBOT --format XML:HTML ${TESTDATADIR}/DocFormat.py ${OUTXML}
+    Copy File    ${OUTXML}    ${OUTPREFIX}-2.xml
+    Test Format In XML:HTML    HTML    lib=${OUTPREFIX}-2.xml
+
+Compare HTML from XML:HTML
+    [Template]    NONE
+    Run Libdoc    -F ROBOT --format XML:HTML ${TESTDATADIR}/DocFormat.py ${OUTPREFIX}_xml_with_html.xml
+    Test Format In HTML    <b>bold</b> or &lt;b&gt;bold&lt;/b&gt; ${EXAMPLE LINK}    
+    ...                    ${EMPTY}    
+    ...                    ${OUTPREFIX}_xml_with_html.xml
```

Comment:
Instead of passing `${EMPTY}` as the second argument, you could use the named argument syntax like `lib=<value>`.

Reply:
done